dataikuapi/dss/ml.py

```diff
@@ -643,6 +660,335 @@ def __repr__(self):
+        :rtype: str
+        """
+        return self._internal_dict["successRatio"]
+
+    @expected_valid_ratio.setter
+    def expected_valid_ratio(self, expected_valid_ratio):
+        self._internal_dict["successRatio"] = expected_valid_ratio
+
+    @property
+    def expected_range(self):
+        """
+        Returns the expected range on which the valid ratio will be calculated
+        :rtype: tuple(float,float)
+        """
+        if "expectedMinValue" in self._internal_dict and "expectedMaxValue" in self._internal_dict:
+            return self._internal_dict["expectedMinValue"], self._internal_dict["expectedMaxValue"]
+        else:
+            return None
+
+    @expected_range.setter
+    def expected_range(self, expected_range):
+        if not isinstance(expected_range, tuple):
+            raise ValueError("Expected range needs to be a tuple")
+        if self.expected_class is not None:
+            raise ValueError("Expected class and expected range can't be both set")
+        if expected_range[0] > expected_range:
+            raise ValueError("Expected range needs to be sorted in ascending order. (min value, max value)")
+        self._internal_dict["expectedMinValue"] = expected_range[0]
+        self._internal_dict["expectedMaxValue"] = expected_range[1]
+
+
+class DSSMLAssertionsMetrics(object):
+    """
+    Object that represents the per assertion metrics for all assertions on a trained model
+    Do not create this object directly, use :meth:`dataikuapi.dss.ml.DSSTrainedPredictionModelDetails.get_per_assertion_metrics()` instead
+    """
+    def __init__(self, data):
+        self._internal_dict = data
+
+    def get_raw(self):
+        """
+        Gets the raw dictionary of the assertions metrics
+        :rtype: dict
+        """
+        return self._internal_dict
+
+    def get_per_assertion_metric(self, assertion_name):
+        """
+        Retrieves the metric computed for this trained model for the assertion with the provided name (or None)
+
+        :param str assertion_name: Name of the assertion
+
+        :returns: an object representing assertion metrics
+        :rtype: :class:`dataikuapi.dss.ml.DSSMLAssertionMetric`
+        """
+        for assertion_metric_dict in self._internal_dict["perAssertion"]:
+            if assertion_name == assertion_metric_dict["name"]:
+                return DSSMLAssertionMetric(assertion_metric_dict)
+        return None
+
+    @property
+    def positive_assertion_ratio(self):
+        """
+        Returns the ratio of passing assertions
+        :rtype: float
+        """
+        return self._internal_dict['positiveAssertionsRatio']
+
+
+class DSSMLAssertionMetric(object):
+    """
+    Object that represents the result of an assertion on a trained model
+    Do not create this object directly, use :meth:`dataikuapi.dss.ml.DSSMLPerAssertionMetrics.get_assertion_metric(self, assertion_name)` instead
```

Comment:
The class is called `DSSMLAssertionsMetrics` (no `Per`). To check: doc method/class names everywhere in new docstrings.